packages/esm-commons-lib/src/api/api.ts

```diff
@@ -179,6 +179,7 @@ export function fetchPatientComputedConcept_HIV_Status(patientUUID: string) {
 }
 
 export function fetchPatientLastEncounter(patientUuid: string, encounterType) {
+  console.log('fetchPatientLastEncounter', patientUuid, encounterType);
```

Comment:
@pirupius You may need to remove this log

Reply:
Definitely. Don't know how i missed that

Reply:
Done